integration_test/third_party_apps_data/applications/mysql/expected_metrics.yaml

```diff
@@ -9,13 +9,29 @@
   kind: GAUGE
   monitored_resource: gce_instance
   labels:
-    kind: data|dirty|flushed|free|misc|total
+    kind: data|free|misc
+- type: workload.googleapis.com/mysql.buffer_pool_data_pages
```

Comment:
Nit: prefer sorting metrics alphabetically by type so that they don't get rearranged when generate_expected_metrics gets run